controllers/ocsinitialization/ocsinitialization_controller.go

```diff
@@ -380,6 +380,11 @@ func (r *OCSInitializationReconciler) ensureOcsOperatorConfigExists(initialData
 			ocsOperatorConfigData[util.CsiRemoveHolderPodsKey] = ocsOperatorConfig.Data[util.CsiRemoveHolderPodsKey]
 		}
 
+		if r.allowRemoteStorageConsumers() {
+			ocsOperatorConfigData[util.CsiEnableCephFSKey] = "false"
+			ocsOperatorConfigData[util.CsiEnableRBDKey] = "false"
```

Comment:
set this new key `ROOK_CSI_DISABLE_DRIVER` to disable csi reconcile by Rook https://github.com/rook/rook/pull/13966

Reply:
Uh oh, I need to raise another pr to rook d/s then 😅

Reply:
i think its not required, it will be part of regular sync once above PR is merged.

Reply:
raised https://github.com/red-hat-storage/rook/pull/600.

Reply:
forward reference w/o pr gettting merged across repos is creating an issue for u/s CI, will track the debt in https://github.com/red-hat-storage/ocs-operator/issues/2526